src/arrays/Arrays.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    public function repeatArrayValues(array $input): array
+    {
+        $arr = [];
+        foreach ($input as $itm) {
```

Comment:
please rewrite this method using php build in array functions

Reply:
Any hint pls?

Reply:
https://www.php.net/manual/en/ref.array.php